bower.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "transportapi.js",
-  "version": "0.0.7",
+  "version": "0.1.0",
   "main": "dist/scripts/transportapi.min.js",
   "description": "A proxy API to TAPI endpoints",
-  "license": "TODO",
+  "license": "",
```

Comment:
CCBY isn't design to apply well to software code. Options are:

* GFDL requires sharealike & attribution
* BSD is more permissive. It requires only attribution + non-endorsement clause
* MIT is more permissive. It requires only attribution. Seems popular with modern OS software.
* Public domain is the most permissive. No requirements. Several similar things by other names: CC0 (offered as an option by github).  WTFPL same but ...a bit rude. "unlicense". A newish license, seems not so well known, but designed to assign your code to the public domain. People can trivially swap between this options anyway of course.

I think 'unlicense' would be my recommendation for transportapi.js   We don't want people to be in a state of worry about blending the library code in with projects under different licenses, or worry about whether they're providing correct credit to us for use of this code. We want developers to grab this little library and "do what the fuck they want with it" without giving code copyright legalities a second thought  (...except we probably want them to think about crediting TransportAPI where they _display the data_. That's a distinct data license/T&Cs issue (which this library can help developers be in compliance with))   Since we're _only_ talking about the copyright status of the javascript code in this library. We should just be super-permissive.